src/auction.js

```diff
@@ -933,3 +933,38 @@ function isValidPrice(bid) {
   if (!maxBidValue || !bid.cpm) return true;
   return maxBidValue >= Number(bid.cpm);
 }
+
+/**
+ * @summary This is the function which will be called to exit module and continue the auction.
+ */
+export function continueAuction(hookConfig, delayedAuctions, callback) {
```

Comment:
to share code between modules, use libraries. core is always included, but this is only needed if you use priceFloors or currency.